src/main/scala/com/gu/zuora/ProductSubscriptionBuilders.scala

```diff
@@ -15,8 +17,9 @@ import scala.util.{Failure, Success, Try}
 
 object ProductSubscriptionBuilders {
 
-  def getProductRatePlanId[PT <: ProductType, P <: Product](product: P, productType: PT): ProductRatePlanId = {
-    val touchpointEnvironment = TouchPointEnvironments.fromStage(Configuration.stage)
+  def getProductRatePlanId[PT <: ProductType, P <: Product](product: P, productType: PT, isTestUser: Boolean): ProductRatePlanId = {
+    val touchpointEnvironment = if (isTestUser) UAT else TouchPointEnvironments.fromStage(Configuration.stage)
```

Comment:
Sorry I should have spotted this during code review last time but it had been a long day!